pkg/execution/state/driver_response.go

```diff
@@ -402,6 +405,12 @@ func (r *DriverResponse) StandardError() StandardError {
 				}
 			}
 		}
+
+		if cause, ok := processed["cause"].(map[string]any); ok {
+			if causeByt, err := json.Marshal(cause); err == nil {
+				_ = json.Unmarshal(causeByt, &ret.Cause)
```

Comment:
Is it possible to do this without marshalling then unmarshalling into a map?  This seems like (at scale) a bunch of throwaway CPU and memory

Reply:
`cause` can now be any value, so we're no longer doing the dance here.